channel/state/test-fixtures.go

```diff
@@ -0,0 +1,27 @@
+package state
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
+	"github.com/statechannels/go-nitro/types"
+)
+
+var chainId, _ = big.NewInt(0).SetString("9001", 10)
+
+var TestState = State{
```

Comment:
I see that this is imported in `/protocols`, so must be an export.

Should globally useful test-fixture code like this live in a text-fixtures package, to avoid diluting the namespace of each package?

(ie, now `state.TestState` is visible everywhere `state` is imported)

Reply:
My naive attempt at this has fallen foul of 
```
package github.com/statechannels/go-nitro/channel/state
	imports github.com/statechannels/go-nitro/fixtures: import cycle not allowed in test`
```

Reply:
Right... Not immediately clear to me if there's a way out of that loop.